test/TensorFlow/crashers.swift

```diff
@@ -1,8 +1,5 @@
 // RUN: %target-swift-frontend -Xllvm -tf-dump-intermediates -O -emit-sil %s -verify
 
-// FIXME(b/78371828): Should this test work with optimized_stdlib?
-// UNSUPPORTED: optimized_stdlib
```

Comment:
will removing this fail the test on OSS side?

same for retain_release.swift below (I'm trying to re-enable that test in the other googleprivate branch of Marc's)

Reply:
The test used to fail on OSS because TensorFlow was optimized. Now that it's not optimized, this test passes on OSS. (Same for the other tests).